core/src/main/scala/cats/data/EitherT.scala

```diff
@@ -91,6 +92,14 @@ final case class EitherT[F[_], A, B](value: F[Either[A, B]]) {
 
   def map[D](f: B => D)(implicit F: Functor[F]): EitherT[F, A, D] = bimap(identity, f)
 
+  def transformF[G[_]](fe: FunctionK[F, G]): EitherT[G, A, B] = EitherT(fe(value))
```

Comment:
In accordance with the corresponding method on `Kleisli` or `ReaderT` being called only `transform`, maybe we should name it this way also? See #1724